src/components/Carousel.js

```diff
@@ -514,11 +522,11 @@ class Carousel extends Component {
     }
 
     render () {
-        if (!this.props.children || this.props.children.length === 0) {
+        if (!this.props.children || React.Children.count(this.props.children) === 0) {
             return null;
         }
 
-        const itemsLength = this.props.children.length;
+        const itemsLength = React.Children.count(this.props.children);
```

Comment:
Please `import React, { Component, Children } from 'react'` and replace all these usages of `React.Children.count` by a less verbose `Children.count`